helm/ph-ee-engine/templates/connector-message-gateway.yaml

```diff
@@ -11,13 +15,13 @@ spec:
               mountPath: /docker-entrypoint-initdb.d/
             - name: message-gateway-mysql-persistent-storage
               mountPath: /var/lib/mysql/
-        volumes:
-        - name: message-gateway-mysql-initdb
-          configMap:
-            name: message-gateway-mysql-initdb
-        - name: message-gateway-mysql-persistent-storage
-          persistentVolumeClaim:
-            claimName: message-gateway-mysql-pv-claim
+          volumes:
```

Comment:
actually, not sure about this part here. it might have satisfied the linter, but not sure it passes an install.